_posts/en/newsletters/2020-08-26-newsletter.md

```diff
@@ -90,22 +90,24 @@ release candidates.*
   to the same address when only a subset of those inputs might be needed.
   For this reason, `avoidpartialspends` is disabled by default unless
   the `avoid_reuse` flag is enabled for the wallet (see [Newsletter
-  #52][news52 avoid_reuse]).  The new `maxapsfee` feature now gives
-  users a choice between three configurations:
+  #52][news52 avoid_reuse]).  It is for this default case that maxapsfee
+  was conceived---its addition gives users a choice between three
+  configurations:
 
-    1. With `-maxapsfee=-1`, partial spend avoidance is completely
+    1. `-maxapsfee=-1`: partial spend avoidance is completely
        disabled to optimize for faster fee calculations, which may be
        useful for very large wallets with many UTXOs.
 
-    2. With `-maxapsfee=0` (the default value), fee calculations are
-       made using both coin selection algorithms, and partial spends are
-       avoided if the fees are less than or equal to regular coin selection.
+    2. `-maxapsfee=0` (the default value): fee calculations are
+       made using both coin selection algorithms.  Whichever result is
+       cheaper is used; if they both result in the same cost, partial
+       spend avoidance is used.
 
-    3. With `maxapsfee` set to greater than `0`, partial spends are
-       avoided if the absolute fee increase is less than or equal to the
-       value passed. For example, `-maxapsfee=0.00001000` means the
-       wallet will avoid partial spends if the absolute fee difference
-       is less than or equal to 1,000 sats.
+    3. `maxapsfee` set to greater than `0`: partial spend avoidance is
+       used whenever the maximum additional cost it adds to the
+       transaction is the passed amount.  For example,
```

Comment:
"whenever the maximum additional cost it adds to the transaction is the passed amount" seems less clear to me than the first and second versions, but I'll defer to any other opinion here